src/helpers/images.ts

```diff
@@ -0,0 +1,6 @@
+import Config from 'react-native-config';
+
+import { images } from 'app/assets';
+
+export const getLogoSource = () =>
+  Config.IS_BETA === '1' ? images.goldWalletLogoBlackBeta : images.goldWalletLogoBlack;
```

Comment:
IS_BETA should be false/true not a number, parse the string from env like `isBeta: JSON.parse(IS_BETA)` in the config/index.js

Reply:
Okay, good that something like this exists, didn't notice